pkg/arvo/sys/vane/jael.hoon

```diff
@@ -35,10 +35,18 @@
 ::  manage subscriptions efficiently.
 ::
 =>  |%
-+$  state                                               ::  all vane state
-  $:  ver=$0                                            ::  vane version
++$  any-state  $%(state-0 state-1)
+::
++$  state-0
+  $:  %0
+      pki=state-pki                                     ::
+      etn=state-eth-node                                ::  eth connection state
+  ==                                                    ::
++$  state-1
+  $:  %1
       pki=state-pki                                     ::
       etn=state-eth-node                                ::  eth connection state
+      step=@ud
```

Comment:
Organizationally, I would put this next to `lyf` in `own.pki`.

Reply:
Done. That had some ripple effect on the changes to state versioning (now `state-pki` is versioned), so might be worth to take another look.